modules/dns_dnscmd/dns_dnscmd_main.rb

```diff
@@ -118,5 +127,30 @@ def enum_zones
       logger.debug "Enumerated dns zones: #{zones}"
       zones
     end
+
+    def get_ptr_ip(zone, ip)
+      ip6 = ip.include?(':')
```

Comment:
> 3. check that passed in address belongs to the subnet (IPAddr#include?)

This one is pretty hard. Can we levarage this on dnscmd? It will fail `DNS_ERROR_NOT_ALLOWED_UNDER_DELEGATION` if we add an IP out of range...